app/my_script.py

```diff
@@ -1,4 +1,265 @@
+                              'SR2', 'Dato', 'Kommentar', 'Unnamed: 25', 'Kabeltype', 'Kontinuer',
+                              '15 min', '1 time', '40 timer', 'Luftledertype']
+
+# Columns in DD20 containing components restriction values
+DD20_COLUMNS_COMPONENT_RESTICT = ['Unnamed: 5', 'AF1', 'LA1', 'SA1', 'TR1', 'Unnamed: 11',
+                                  'SR1', 'Unnamed: 15', 'AF2', 'LA2', 'SA2', 'TR2', 'Unnamed: 21', 'SR2']
+
+# Columns in DD20 defined as constants, since data will be extracted from them
+DD20_LINENAME_COL_NM = 'Linjenavn'
+DD20_KV_COL_NM = 'Spændingsniveau'
+DD20_CONDUCTOR_TYPE_COL_NM = 'Luftledertype'
+DD20_CONTINIOUS_COL_NM = 'Kontinuer'
+DD20_15M_COL_NM = '15 min'
+DD20_1H_COL_NM = '1 time'
+DD20_40H_COL_NM = '40 timer'
+
+
+def convert_voltage_level_to_letter(voltage_level: float) -> str:
+    """Converts voltage level to voltage letter representation.
+
+    Parameters
+    ----------
+    voltage_level : float
+        Voltage level in kV.
+
+    Returns
+    -------
+    str
+        Voltage letter.
+
+    Example
+    -------
+    convert_voltage_level_to_letter(400) return 'C'
+    """
+    if voltage_level >= 420:
+        voltage_letter = 'B'
+    elif 380 <= voltage_level < 420:
+        voltage_letter = 'C'
+    elif 220 <= voltage_level < 380:
+        voltage_letter = 'D'
+    elif 110 <= voltage_level < 220:
+        voltage_letter = 'E'
+    elif 60 <= voltage_level < 110:
+        voltage_letter = 'F'
+    elif 45 <= voltage_level < 60:
+        voltage_letter = 'G'
+    elif 30 <= voltage_level < 45:
+        voltage_letter = 'H'
+    elif 20 <= voltage_level < 30:
+        voltage_letter = 'J'
+    elif 10 <= voltage_level < 20:
+        voltage_letter = 'K'
+    elif 6 <= voltage_level < 10:
+        voltage_letter = 'L'
+    elif 1 <= voltage_level < 6:
+        voltage_letter = 'M'
+    elif voltage_level < 1:
+        voltage_letter = 'N'
+
+    return voltage_letter
+
+
+def parse_excel_sheet_to_dataframe(file_path: str, sheet_name: str, header_index: int = 0) -> pd.DataFrame:
+    """Read sheet from excel file and parse it to pandas dataframe.
+
+    Parameters
+    ----------
+    file_path : str
+        Full path of the excel file.
+    sheet_name : str
+        Name of sheet in excel file.
+    header_index : int
+        Index number for row to be used as header (Default = 0)
+
+    Returns
+    -------
+    pd.DataFrame
+        A dataframe containing the data from excel sheet.
+    """
+    # try to read data from excel file to dataframe.
+    try:
+        dataframe = pd.read_excel(io=file_path, sheet_name=sheet_name, header=header_index)
+        log.info(f"Excel data from sheet '{sheet_name}' in '{file_path}' was parsed to dataframe.")
+    except Exception as e:
+        log.exception(f"Parsing data from sheet: '{file_path}' in excel file: '{sheet_name}' failed with message: '{e}'.")
+        sys.exit(1)
```

Comment:
I would suggest using 'raise' in stead of manually exiting. That way you don't have to import the sys module for just exiting, and the 'raise' will call a system exit anyway if unhandled. But the 'clever' thing is that i can actually be handled higher up in the stack. That way you will be using the built-in error/exception system and your function will survive being called within a 'try/except' in case someone actually wants to handle any errors "higher up".

```suggestion
        raise
```
or 
````suggestion
        raise IOError("File not found or incorrectly formatted") from e
````

The above suggestions are examples of how it COULD be done - ONE of them should be selected, or a similar approach. The first suggestion will just re-raise the exception that happened - the second suggestion will add another exception to the stack (giving more information, generally).